proto-grpc.md

```diff
@@ -0,0 +1,254 @@
+# Protobuf and gRPC with Bazel
+
+The use of [Protobuf](https://developers.google.com/protocol-buffers) and
+[gRPC](https://grpc.io) with [Bazel](https://bazel.build) is a particularly common scenario, since
+these three technologies provide a complimentary base of build, serialisation and remote-call upon
+which to build complex systems and products. This document summarises the current state of the
+Protobuf/gRPC Bazel ecosystem and gives advice on getting started when you are looking to integrate
+one of these tools into your project.
+
+Given that all three of Bazel, gRPC and Protobuf are Google-derived projects, you may expect that
+there is a simple integration between them. However, the existence of this document should indicate
+that there is not a single simple answer to integrating these tools. Originally, Protobuf and gRPC
```

Comment:
nit: how did you soft-wrap the document? Maybe we should use a markdown formatter like Prettier so that users don't have to re-flow lines as they make edits, or setup some soft-wrap-aware editor to make changes.

Reply:
I just kept it rough-wrapped whilst writing, then manually fixed it up when I was done. A proper tool for that would be useful though, as I've never found a satisfactory and trustworthy MD extension for vscode